### ISSUE TITLE
Trainer Config in WiC_slicing_tutorial

### ISSUE THREAD

**Issue by honorinli151** [HazyResearch/snorkel-superglue#23]
Hi,

I am trying the WiC_slicing_tutorial, however, I could not reproduce your results with the default trainer config in the notebook.

Could you please share the Trainer config you used to train your pretrained "WiC_slice_verb.pth"?

Thanks

**Comment by vincentschen** [comment]
Hi @honorinli151 — I don't believe we did too special here with the configuration; what parameters are you using?

**Comment by honorinli151** [comment]
The config we are using:
<details>
<summary>Details</summary>

{'seed': None,
 'n_epochs': 5,
 'train_split': 'train',
 'valid_split': 'valid',
 'test_split': 'test',
 'progress_bar': True,
 'model_config': {'model_path': None, 'device': 0, 'dataparallel': True},
 'log_manager_config': {'counter_unit': 'epochs', 'evaluation_freq': 1.0},
 'checkpointing': False,
 'checkpointer_config': {'checkpoint_dir': None,
  'checkpoint_factor': 1,
  'checkpoint_metric': 'model/all/train/loss:min',
  'checkpoint_task_metrics': None,
  'checkpoint_runway': 0,
  'checkpoint_clear': True},
 'logging': False,
 'log_writer_config': {'writer': 'tensorboard',
  'log_root': 'logs',
  'run_name': None},
 'optimizer_config': {'optimizer': 'adam',
  'lr': 0.001,
  'l2': 0.0,
  'grad_clip': 1.0,
  'sgd_config': {'momentum': 0.9},
  'adam_config': {'betas': (0.9, 0.999), 'amsgrad': False},
  'adamax_config': {'betas': (0.9, 0.999), 'eps': 1e-08}},
 'lr_scheduler_config': {'lr_scheduler': 'constant',
  'warmup_steps': 0,
  'warmup_unit': 'batches',
  'warmup_percentage': 0.0,
  'min_lr': 0.0,
  'exponential_config': {'gamma': 0.9},
  'plateau_config': {'factor': 0.5, 'patience': 10, 'threshold': 0.0001}},
 'task_scheduler': 'shuffled'}

</details>

The exact notebook could be found [here](https://github.com/HazyResearch/snorkel-superglue/files/3966200/WiC_slicing_tutorial.zip) in attachment.

**Comment by vincentschen** [comment]
So sorry about missing this @honorinli151!

Are you still experiencing issues here?

If so, given your previous config, it looked like the model wasn't generalizing — can you try training for more epochs? I'd also suggest fine-tuning the slice tasks after first training a the stable baseline model.
